algebra/ring.lean

```diff
@@ -184,6 +184,17 @@ instance integral_domain.to_nonzero_comm_ring (α : Type*) [id : integral_domain
   nonzero_comm_ring α :=
 { ..id }
 
+lemma semiring.zero_of_zero_eq_one {α} [semiring α] (h01 : (0 : α) = 1) (a : α) : a = 0 :=
+by rw [←one_mul a,←h01,zero_mul]
```

Comment:
Add a space after each comma (this is a general rule, only avoid the space when at the end of a line)